packages/aws-cdk/lib/init-templates/jsii/info.json

```diff
@@ -0,0 +1,4 @@
+{
+    "description": "Template for a CDK Construct Library",
+    "aliases": "jsii"
```

Comment:
Not sure that's the best alias/name for this template. It does not speak to the actual user/business value, merely hints at the technology choice.